src/types/connector.ts

```diff
@@ -71,3 +71,13 @@ export type Connector<TRendererOptions, TConnectorParams> = <TWidgetParams>(
    */
   unmountFn?: Unmounter
 ) => WidgetFactory<TConnectorParams, TWidgetParams>;
+
+/**
+ * Transforms the given items.
+ */
+export type TransformItems<TItem> = (objects: TItem[]) => TItem[];
```

Comment:
Is everyone ok with this name? I can also make it ItemsTransformer or something.

Reply:
@francoischalifour Actually, if you return a item that has the shape ( extends ) it will work too.